peer/main_test.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright IBM Corp. 2016 All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+		 http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"testing"
+)
+
+func TestMain(t *testing.T) {
+	t.Skip("No needs to test main since it is covered by other tests")
+}
```

Comment:
Hi Binh,

I realize from your PR Description comment that the goal of skipping the tests was intentional, and thus justifies why this file is here.  What's not clear is why this is needed.  I am more curious than anything, but perhaps a comment in this file is a good idea as I suspect I won't be the only one wondering what this accomplishes.